extensions/sql-database-projects/src/models/options/deployOptionsModel.ts

```diff
@@ -7,251 +7,82 @@ import * as mssql from 'mssql';
 
 export class DeployOptionsModel {
 	public deploymentOptions: mssql.DeploymentOptions;
-
-	public optionsLookup: Record<string, boolean> = {};
-	public optionsMapTable: Record<string, mssql.DacDeployOptionPropertyBoolean> = {};
+	public booleanOptionsMap: { [key: string]: mssql.DacDeployOptionPropertyBoolean } = {};
+	public optionsLabels: string[] = [];
+	public optionsNameAndPropMap: { [key: string]: string } = {};
+	public optionsValueLookup: { [key: string]: boolean } = {};
 
 	constructor(defaultOptions: mssql.DeploymentOptions) {
 		this.deploymentOptions = defaultOptions;
```

Comment:
Just do this in the constructor `private deploymentOptions: mssql.DeploymentOptions`